src/Server/Program.cs

```diff
@@ -100,18 +142,14 @@ public Program(string[] args)
                     services.AddSingleton<DicomWebExportService>();
                     services.AddSingleton<IInstanceStoredNotificationService, InstanceStoredNotificationService>();
                     services.AddSingleton<IApplicationEntityManager, ApplicationEntityManager>();
-                    services.AddSingleton<IJobStore, JobStore>();
-                    services.AddSingleton<IInferenceRequestStore, InferenceRequestStore>();
+                    services.AddSingleton<IClaraAeChangedNotificationService, ClaraAeChangedNotificationService>();
 
                     services.AddHttpClient("dicomweb", configure => configure.Timeout = TimeSpan.FromMinutes(60))
                         .SetHandlerLifetime(TimeSpan.FromMinutes(60));
```

Comment:
I feel like this 60 minute lifetime value should be a named `static readonly` someplace, because it seems important.

also does this line need to align with the line above? if so, definitely make them a `static readonly`

Reply:
Fixed 🙏